p/invg/products.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+module.exports = [
+	// https://github.com/public-transport/hafas-client/issues/93#issuecomment-437868265
+	{
+		id: 'bus',
+		mode: 'bus',
+		bitmasks: [1],
+		name: 'Bus',
+		short: 'Bus',
+		default: true // the other `bus` has `false`
+	},
+	{
+		id: 'express-train',
+		mode: 'train',
+		bitmasks: [2],
+		name: 'High-speed train',
+		short: 'Train',
+		default: false
+	},
+	{
+		id: 'regional-train',
+		mode: 'train',
+		bitmasks: [4],
+		name: 'Regional train',
+		short: 'Train',
+		default: false
+	},
+	{
+		id: 'local-train',
+		mode: 'train',
+		bitmasks: [8],
+		name: 'Nahverkehrszug',
+		short: 'Zug',
+		default: true
+	},
+	{
+		id: 'bus-2',
```

Comment:
The naming here seems sub-optimal 😛 IMHO we should rather group both bus products if we don't exactly know the difference between them.

Reply:
I think the reason why I didn't merge them back then was that one is on by default, the other is not. Will merge them nevertheless.